hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -675,6 +679,7 @@
               </execution>
             </executions>
           </plugin>
+-->
```

Comment:
It looks like there is no need to update this pom file, if that is true can you reset the changes in this pom?